bundles/org.eclipse.equinox.slf4j.stub/build.properties

```diff
@@ -15,3 +15,4 @@ bin.includes = META-INF/,\
                .,\
                about.html
 src.includes = about.html
+pom.model.property.skipAPIAnalysis=true
```

Comment:
This should not be necessary anymore.
The api-check profile from the parent-platform now 'computes' `skipAPIAnalysis` based on the presence of the API-builder.